.eslintrc.json

```
@@ -55,8 +55,8 @@
           "warn",
           {
             "ignoreRestSiblings": true,
-            "argsIgnorePattern": "^_([0-9])?$",
-            "varsIgnorePattern": "^_([0-9])?$"
+            "argsIgnorePattern": "^_[a-zA-Z0-9]*$",
+            "varsIgnorePattern": "^_[a-zA-Z0-9]*$"
```

```suggestion
            "argsIgnorePattern": "^\w+$",
            "varsIgnorePattern": "^\w+$"
```

---

```suggestion
            "argsIgnorePattern": "^_[a-zA-Z]\w*$",
            "varsIgnorePattern": "^_[a-zA-Z]\w*$"
```
This might be better if we want to prohibit e.g. `_1`, `_2`, etc.

---

`\w` is not the same as `[a-zA-Z0-9]` since it allows `_`, which is not in line with our naming standards:
`_foo_bar` should be flagged as incorrect.